### PR TITLE
Add support for azure blob storage

### PR DESCRIPTION
**Pull request by elsesiy**
Integrate fluent-plugin-azure-storage-append-blob

### CODE REVIEW

**Comment by elsesiy** [comment]
@repeatedly I just updated this PR for 1.9.x, when can we merge this? Thanks!

**Comment by repeatedly** [comment]
Currently, we don't add new image to this repository. See: https://github.com/fluent/fluentd-kubernetes-daemonset/issues/293
If you publish the image to your repository, we can add the link to README.

File: README.md [hunk 1]
```diff
@@ -168,7 +169,7 @@ oc patch ds fluentd -p "spec:
       - name: fluentd
         securityContext:
           privileged: true"
-oc delete pod -l k8s-app=fluentd-logging
+oc delete pod -l k8s-app = fluentd-logging
```
Comment (cosmo0920) [inline README.md:172]: Spaces around equal causes oc delete pod command failure.
Cloud you delete them?

**Comment by repeatedly** [comment]
Add v1-debian-azureblob with automated build.

**Comment by cosmo0920** [comment]
:tada:

**Pull request merged**